src/jbi/whiteboard_actions/default.py

```diff
@@ -6,35 +6,130 @@
 `init` should return a __call__able
 """
 
+from src.app.environment import get_settings
+from src.jbi.bugzilla_objects import BugzillaBug, BugzillaWebhookRequest
+from src.jbi.model import ActionError
+from src.jbi.service import get_bugzilla, get_jira
+
 
 def init(whiteboard_tag, jira_project_key, **kwargs):
-    """
-    Function that takes required and optional params and returns a callable object
-    :param whiteboard_tag: Required param
-    :param jira_project_key: Required param
-    :param kwargs: Optional params
-    :return: DefaultExecutor
-    """
+    """Function that takes required and optional params and returns a callable object"""
     return DefaultExecutor(
         whiteboard_tag=whiteboard_tag, jira_project_key=jira_project_key, **kwargs
     )
 
 
 class DefaultExecutor:
-    """
-    Callable class that encapsulates the default action
-    """
+    """Callable class that encapsulates the default action."""
 
     def __init__(self, **kwargs):
         """Initialize DefaultExecutor Object"""
         self.parameters = kwargs
         self.whiteboard_tag = kwargs.get("whiteboard_tag")
         self.jira_project_key = kwargs.get("jira_project_key")
+        self.bugzilla_client = get_bugzilla()
+        self.jira_client = get_jira()
+        self.settings = get_settings()
+
+    def __call__(  # pylint: disable=too-many-locals,too-many-return-statements,inconsistent-return-statements
+        self, payload: BugzillaWebhookRequest
+    ):
+        """Called from BZ webhook when default action is used. All default-action webhook-events are processed here."""
+        target = payload.event.target
+        if not payload.bug or not isinstance(payload.bug, BugzillaBug):
+            raise ActionError("payload is expected to have bug data")
+
+        if target == "comment":
+            self.comment_create_or_noop(payload=payload)
+        if target == "bug":
+            self.bug_create_or_update(payload=payload)
```

Comment:
Something seems wrong here since you don't use the returned values of these functions